buildpipeline/windows.groovy

```diff
@@ -0,0 +1,93 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Config - Build configuration. Note that we don't using 'Configuration' since it's used
+//          in the build scripts and this can cause problems.
+// OuterLoop - If true, runs outerloop, if false runs just innerloop
+
+def submittedHelixJson = null
+
+simpleNode('Windows_NT','latest') {
+    stage ('Checkout source') {
+        retry (10) {
+            checkout scm
+        }
+    }
+
+    def logFolder = getLogFolder()
+    def framework = ''
+    if (params.TargetGroup == 'all') {
+        framework = '-allConfigurations'
+    }
+    else {
+        framework = '-framework:${params.TargetGroup}'
+    }
+    def archGroup = 'x64' // Should parameterize in the future.
+
+    def submitToHelix = (params.TargetGroup == 'netcoreapp')
+
+    stage ('Clean') {
+        bat '.\\clean.cmd -all'
+    }
+    stage ('Sync') {
+        bat '.\\sync.cmd -p -- /p:ArchGroup=${archGroup} /p:RuntimeOS=win10'
+    }
+    stage ('Generate Version Assets') {
+        bat '.\\build-managed.cmd -GenerateVersion'
+    }
+    stage ('Build Product') {
+        bat ".\\build.cmd ${framework} -buildArch=${archGroup} -${params.Config} -- /p:SignType=real /p:RuntimeOS=win10"
+    }
+    stage ('Build Tests') {
+        def additionalArgs = ''
+        def archiveTests = 'false'
+        if (params.OuterLoop) {
+            additionalArgs += '-Outerloop'
```

Comment:
maybe dumb question, how do whitespace characters get injected between all these concatenations of additional args? It looks like it could easily end up being  "-Outerlopp-SkipTests-SkipTests"...